src/logic/calculate.js

```diff
@@ -29,8 +29,11 @@ export default function calculate(obj, buttonName) {
     // If there is an operation, update next
     if (obj.operation) {
       if (obj.next && obj.next !== '0') {
+        console.log('WTF1');
         return { ...obj, next: obj.next + buttonName };
       }
+      console.log('WTF2');
+
```

Comment:
- [ ] [REQUIRED] Please, try to remove any `console.log` from the production code as a best practice.

> **Note:** Kindly, try to keep your repo code as professional as possible (e.g: avoid use of explicit language) as there is a potential it can be seen by recruiters.

Reply:
@OmarRamoun  This is embarrassing 
I should've removed it, no one deserves to see this 😂
I just was debugging and forgot. 
Thank you 😊